index.html

```diff
@@ -804,7 +804,7 @@ <h3 class="title-wrapper center card-header"><a href="https://github.com/ranelpa
                         this.isInputCorrect = true;
                         this.hitsCorrect += 1;
                     }
-                    else {
+                    else if (this.expectedPhrase !== typedPhrase.replace(/ +$/, "")) {
```

Comment:
Please use `.trimEnd()` which is simpler/more readable:
`else if (this.expectedPhrase !== typedPhrase.trimEnd()) {`